lib/Vimscript/Builtins.idr

```diff
@@ -91,3 +103,163 @@ strpart' str start = unsafePerformIO
 ||| ```
 expand : String -> VIM_IO String
 expand str = builtin "expand" (String -> VIM_IO String) str 
+
+cursor : (line : Int) -> (col : Int) -> VIM_IO ()
+cursor line col = builtin "cursor" (Int -> Int -> VIM_IO ()) line col
+
+getcwd : VIM_IO String
+getcwd = builtin "getcwd" (VIM_IO String)
+
+||| Execute a shell command and discard the output.
+system' : String -> VIM_IO ()
```

Comment:
Maybe a Haskell naming scheme for these, i.e. `system_`?

Reply:
Good idea.

Reply:
Reminder for this one. :)

Reply:
Done!

Reply:
Fixed in https://github.com/owickstrom/idris-vimscript/pull/1/commits/243984ff026a9a81b90d5f013521921dec100d1f